api.md

```diff
@@ -3406,138 +3312,143 @@ SELECT pg_size_pretty(after_compression_total_bytes) as total
-```
-The expected output:
-```
- table_bytes | index_bytes | toast_bytes | total_bytes
--------------+-------------+-------------+-------------
-  1227661312 |  1685979136 |      180224 |  2913820672
+-- disttable is a distributed hypertable --
+SELECT * FROM hypertable_detailed_size('disttable') ORDER BY node_name;
+
+ table_bytes | index_bytes | toast_bytes | total_bytes |  node_name
+-------------+-------------+-------------+-------------+-------------
+       16384 |       32768 |           0 |       49152 | data_node_1
+        8192 |       16384 |           0 |       24576 | data_node_2
+
 ```
 ---
 
-## hypertable_relation_size_pretty() [](hypertable_relation_size_pretty)
+## chunks_detailed_size()  :community_function: [](chunks_detailed_size)
 
-Get relation size of hypertable like `pg_relation_size(hypertable)`.
+Get per chunk size information of hypertable.
+This function will return multiple rows of output for a single
+chunk, if this is a distributed hypertable. Meta data for a chunk can be
+viewed using `timescaledb_information.chunks` view.
 
-#### Required Arguments [](hypertable_relation_size_pretty-required-arguments)
+#### Required Arguments [](chunks_detailed_size-required-arguments)
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of hypertable to get relation size for.|
+| `main_table` | Name of the hypertable|
 
-#### Returns [](hypertable_relation_size_pretty-returns)
+#### Returns [](chunks_detailed_size-returns)
 |Column|Description|
 |---|---|
-|table_size|Pretty output of table_bytes|
-|index_size|Pretty output of index_bytes|
-|toast_size|Pretty output of toast_bytes|
-|total_size|Pretty output of total_bytes|
+|chunk_schema| Schema name of the chunk |
+|chunk_name| Name of the chunk|
+|table_bytes|Disk space used by the chunk table|
+|index_bytes|Disk space used by indexes|
+|toast_bytes|Disk space of toast tables|
+|total_bytes|Total disk space used by the chunk, including all indexes and TOAST data|
+|node_name| Node for which size is reported, applicable only to distributed hypertables|
 
-#### Sample Usage [](hypertable_relation_size_pretty-examples)
-```sql
-SELECT * FROM hypertable_relation_size_pretty('conditions');
-```
-or, to reduce the output, a common use is:
+#### Sample Usage [](chunks_detailed_size-examples)
 ```sql
-SELECT table_size, index_size, toast_size, total_size
-FROM hypertable_relation_size_pretty('conditions');
+SELECT * FROM chunks_detailed_size('dist_table')
+ORDER BY chunk_name, node_name;
+     chunk_schema      |      chunk_name       | table_bytes | index_bytes | toast_bytes | total_bytes |       node_name
+-----------------------+-----------------------+-------------+-------------+-------------+-------------+-----------------------
+ _timescaledb_internal | _dist_hyper_1_1_chunk |        8192 |       32768 |           0 |       40960 | db_node1
+ _timescaledb_internal | _dist_hyper_1_1_chunk |        8192 |       32768 |           0 |       40960 | db_node2
```

Comment:
See my question above about why a chunk appears on multiple data nodes?  Given that (at least for now) we don't recommend replication_factor > 1, if that's the only case for it, I would simplify this example to show replication_factor = 1.  We can extend these docs once we launch more complete support for replication.

Reply:
Modified. I wasn't aware that we don't recommend this usage.